File: brian_ortega_delahoz/Clases/Programación/Mayo2024/27-May-24/rentaAnual.rb
puts "Ingrese su cantidad de renta anual:"
renta_anual = gets.chomp.to_f

if renta_anual <= 10
    ren = (renta_anual*20)/100
    resul = renta_anual - ren
     puts "Su valor a pagar con el descuento aplicado es de: #{resul}"
     
else
    if 
        renta_anual <= 100
        ren = (renta_anual*10)/100
        resul = renta_anual - ren
         puts "Su valor a pagar con el descuento aplicado es de: #{resul}"
    else
        if 
            renta_anual <= 1000
            ren = (renta_anual*5)/100
            resul = renta_anual - ren
             puts "Su valor a pagar con el descuento aplicado es de: #{resul}"
        else
            if 
                renta_anual > 1000
                ren = (renta_anual*1)/100
                resul = renta_anual - ren
                 puts "Su valor a pagar con el descuento aplicado es de: #{resul}"
            else
            end
        end
    end
end

File: brian_ortega_delahoz/Clases/Programación/Mayo2024/27-May-24/rentaAnual.dart
import 'dart:io';

void main() {
  print("Ingrese su renta anual");
  int renta_anual = int.parse(stdin.readLineSync()!);

  if (renta_anual <= 10) {
    double op = (renta_anual * 20) / 100;
    double resul = renta_anual - op;
    print('Su valor a pagar con el descuento aplicado es de: $resul');
  } else {
    if (renta_anual <= 100) {
      double op = (renta_anual * 10) / 100;
      double resul = renta_anual - op;
      print('Su valor a pagar con el descuento aplicado es de: $resul');
    } else {
      if (renta_anual <= 1000) {
        double op = (renta_anual * 5) / 100;
        double resul = renta_anual - op;
        print('Su valor a pagar con el descuento aplicado es de: $resul');
      } else {
        if (renta_anual > 1000) {
          double op = (renta_anual * 1) / 100;
          double resul = renta_anual - op;
          print('Su valor a pagar con el descuento aplicado es de: $resul');
        }
      }
    }
  }
}
